uber/server.py

```diff
@@ -101,11 +101,17 @@ def index(self):
 cherrypy.tree.mount(Root(), c.PATH, c.APPCONF)
 static_overrides(join(c.MODULE_ROOT, 'static'))
 
-DaemonTask(check_unassigned, interval=300,          name="mail unassg")
-DaemonTask(detect_duplicates, interval=300,         name="mail dupes")
-DaemonTask(check_placeholders, interval=300,        name="mail placeh")
```

Comment:
I moved all of these separate registration checks into a single `DaemonTask`. There's no reason why they can't be executed serially in a single thread, instead of in parallel in multiple threads.

Also, I bumped the interval up to every six hours, because these checks are just generating daily reports anyway. No reason they need to run every five minutes.

Also, I added a 10 minute delay before they start. This gives the `AutomatedEmail` cache (which these tasks use) time to fully populate.